doc/0000-ccl_note/main.tex

```diff
@@ -550,162 +448,213 @@ \subsubsection{Cosmic emulator}
+As in the previous section, the power spectrum at small wavenumber is extrapolated
+using a power-law. This extrapolation is performed below a fiducial value of
+{\tt K\_MIN\_SPLINE} that coincides with the smallest wavenumber output by
+{\tt CLASS}, as in the case of the nonlinear power spectrum described above.
 
-We have found that changing {\tt N$\_$A} to $200$, or changing the sampling of the wavenumber to $5000$ points, does not change the results presented in Figures \ref{fig:NLextrapol} in this section.
-
-\subsubsection{Wishlist for the future}
-\label{Pk_wishlist}
-We plan to implement the following power spectrum methods in the future:
-\begin{itemize}
- \item CAMB,
- \item other emulators,
- \item HOD.
-\end{itemize}
+We have found that changing the sampling in scale factor to $200$, or changing
+the sampling of the wavenumber to $5000$ points, does not change the results
+presented in Figure~\ref{fig:NLextrapol}.
 
 
 \subsubsection{Normalization of the power spectrum}
 \label{sec:PSnorm}
 
-There are two alternative schemes for normalization of the matter power spectrum. The first one is to specify the value of $A_s$, the amplitude of the primordial power spectrum, which is passed directly to {\tt CLASS}. This option is available in the case of the linear/nonlinear matter power spectrum implementation. For these, as well as for BBKS and E\&H transfer functions, there is the additional option to set the normalization of the matter power spectrum by specifying $\sigma_8$, the RMS density contrast averaged over spheres of radius $8h^{-1}$Mpc. The computation of $\sigma_8$ is described in Section \ref{sec:hmf}.
-
-In practice, there is only one argument that encodes the normalization. This is the argument ${\tt norm\_pk}$, which can be passed the power spectrum normalization parameterized by $\sigma_8$ or $A_\mathrm{s}$. As noted above, {\tt ccl$\_$parameters$\_$create} switches to $\sigma_8$ normalization if ${\tt norm\_pk} > 10^{-5}$, and to $A_{\mathrm s}$ normalization otherwise.
-
-In the {\tt python} implementation, {\tt CCL} allows for either $\sigma_8$ or {\tt A\_s} to be passed as parameters.
+There are two schemes for normalization of the matter power spectrum. The first
+one is to specify the value of $A_s$, the amplitude of the primordial power
+spectrum, which is passed directly to {\tt CLASS}. Alternatively, there is the
+option to set the normalization of the matter power spectrum by specifying
+$\sigma_8$, the RMS density contrast averaged over spheres of radius
+$8h^{-1}$Mpc. Note that for linear power spectrum models where {\tt CLASS} is not
+used, only the option to specify $\sigma_8$ is supported. The computation of
+$\sigma_8$ is described in Section \ref{sec:hmf}.
 
 \subsection{Angular power spectra}
```

Comment:
Do we say anywhere that the user can pass a generalized tracer now?

Reply:
Not really the point of these docs. That is an implementation detail.